client/src/menu/game/gameScreenContent/RoleSpecificMenus/SmallOjoMenu.tsx

```diff
@@ -0,0 +1,75 @@
+import { ReactElement } from "react"
+import { Role } from "../../../../game/roleState.d"
+import React from "react"
+import RoleDropdown from "../../../../components/RoleDropdown"
+import GAME_MANAGER from "../../../.."
+import translate from "../../../../game/lang"
+
+export type OjoAction = {
+    type: "none"
+} | {
+    type: "kill",
+    role: Role
+} | {
+    type: "see",
+    role: Role
+}
+
+export default function SmallOjoMenu(props: {action: OjoAction}): ReactElement {
+
+    const sendAction = (action: OjoAction) => {
+        GAME_MANAGER.sendSetOjoAction(action);
+    }
+
+    let actionTypeDropdown = <ActionTypeDropdown action={props.action} onChange={(a)=>{sendAction(a)}}/>
+
+    let roleDropdown = null; 
+    if(props.action.type !== "none"){
+        roleDropdown = <RoleDropdown value={props.action.role} onChange={(role)=>{
+            if(props.action.type === "none") return;
+            sendAction({...props.action, role: role})
+        }}/>;
+    }
+
+    switch(props.action.type){
+        case "none":
+            return <>{actionTypeDropdown}</>
+        case "kill":
+            return <>{actionTypeDropdown}{roleDropdown}</>
+        case "see":
+            return <>{actionTypeDropdown}{roleDropdown}</>
+    }
+}
```

Comment:
This entire function can be inlined into the return statement

Reply:
Also there is no text here. As a player I don't know why my player list has this thing on top of it.

Reply:
Can you help me with this. This also needs to be fixed for amnesiac.